### ISSUE TITLE
adapt-youtube & adapt-vimeo iframes have no title attributes

### ISSUE THREAD

**Issue by simondate** [adaptlearning/adapt_framework#3301]
### Subject of the issue/enhancement/features

These components have no `title` attributes in their iframes. This should be here for screenreaders and is present in other 3rd party [iframe components](https://github.com/LearningPool/adapt-contrib-responsiveIframe/blob/master/templates/responsiveIframe.hbs#L6)

### Screenshots (if you can)

![image](https://user-images.githubusercontent.com/1676635/166438731-acd51a31-a137-4f41-8f24-0e335902414b.png)

**Comment by oliverfoster** [comment]
Use aria-label instead, title is ambiguous as it doesn't behave the same way on iframes and other elements.

**Comment by simondate** [comment]
This seems to be closely tied to this https://github.com/adaptlearning/adapt_framework/issues/2822
